### ISSUE TITLE
SELECT index predis

### ISSUE THREAD

**Issue by quah6o** [Netflix/dynomite#647]
Hi,

we played around with dynomite and would like to use it in our application.
We are using the predis client. Predis sends a initial SELECT 0 which breaks the connection to dynomite. Our application only uses the redis database 0 and we would like to return OK / ignore the select and continue. I am not that familiar with C. Can anyone give me a hint what we have to change to achieve that?

Best regards

**Comment by mcouillard** [comment]
Yeah, Lettuce does this, too, I'm afraid. More about it here: https://github.com/Netflix/dynomite/issues/551

**Comment by ipapapa** [comment]
You could just add the `SELECT` command in the following [parser](https://github.com/Netflix/dynomite/blob/dev/src/proto/dyn_redis.c) and should be fine. If you plan to do that just add a comment with the corresponding limitations.

**Comment by arocki7** [comment]
We are having similar issues. Tried to use Dynomite along with 3Scale and Select Index failed.
